plugins/Denizen/scripts/cooking/allergies.dsc

```diff
@@ -0,0 +1,77 @@
+allergies:
+    type: world
+    debug: false
+    events:
+        on player dies:
+            - if <util.random_chance[12]>:
+                - define allergy <script[allergy_info].data_key[valid_allergies].keys.random>
+                - define effect <script[allergy_info].data_key[valid_effects].keys.random>
+                - flag player allergic:<[allergy]>
+                - flag player allergy_effect:<[effect]>
+                - stop
```

Comment:
Put the last two lines of this script (the flag removals) into an `else` statement and remove this `stop`. This will make it look a bit better.